windows-driver-docs-pr/stream/dmft-design.md

```diff
@@ -255,7 +257,7 @@ Device MFTs must use the following messages to inform DTM about the availability
 
 ### Thread Requirements
 
-Device MFT must not create its own threads. Instead it must use MF Work Queues, whose ID is passed through the [IMFRealtimeClientEx](/windows/win32/api/mfidl/nn-mfidl-imfrealtimeclientex) interface. This is to make sure that all the threads running in the Device MFT gets the correct priority at which the capture pipeline is running. Otherwise it may cause thread priority inversions.
+Device MFT must not create its own threads. Instead it must use MF Work Queues, whose ID is passed through the [IMFRealTimeClientEx](/windows/win32/api/mfidl/nn-mfidl-imfrealtimeclientex) interface. This is to make sure that all the threads running in the Device MFT gets the correct priority at which the capture pipeline is running. Otherwise it may cause thread priority inversions.
```

Comment:
> whose ID is pass [](http://example.com/codeflow?start=79&length=17)

since you are changing this- can we improve the sentence here?.

Intead it must use MF WorkQueue ([https://docs.microsoft.com/en-us/windows/win32/medfound/work-queues](https://docs.microsoft.com/en-us/windows/win32/medfound/work-queues)) which are allocated based on the ID passed to the DMFT through the [IMFRealTimeClientEX]......

Reply:
thx - updated text.